corehq/apps/analytics/management/commands/update_hubspot_properties.py

```diff
@@ -4,7 +4,7 @@
 from django.core.management import BaseCommand
 from django.core.management.base import CommandError
 from corehq.apps.analytics.signals import _get_domain_membership_properties, _get_subscription_properties_by_user
-from corehq.apps.analytics.tasks import _batch_track_on_hubspot
+from corehq.apps.analytics.tasks import _batch_track_on_hubspot, _get_ab_test_properties
```

Comment:
in general prefixing a function name with an underscore is a python convention that it's meant to be "private" to that module/class, so if these are getting imported they should probably also be/have been renamed